src/chrome/content/rules/Wiz.cn.xml

```diff
@@ -26,12 +23,14 @@
 	<target host="api.wiz.cn" />
 	<target host="as.wiz.cn" />
 		<test url="http://as.wiz.cn/wizas/xmlrpc" />
+	<target host="bbs.wiz.cn" />
 	<target host="blog.wiz.cn" />
 	<target host="cdn.wiz.cn" />
-	<target host="get.wiz.cn" />
-	<!-- Find get.wiz.cn tests at: https://www.wiz.cn/downloads-windows.html -->
-	<target host="s2.wiz.cn" />
+	<target host="cdn2.wiz.cn" />
+	<target host="get.wiz.cn" /><!-- Find get.wiz.cn tests at: https://www.wiz.cn/downloads-windows.html -->
 	<target host="url.wiz.cn" />
 
+	<rule from="^http://wiz\.cn/" to="https://www.wiz.cn/" />
```

Comment:
Why is this required?

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/418257579#L599

Reply:
Please add a hint in the comments.